includes/data-stores/class-wc-product-variation-data-store-cpt.php

```diff
@@ -85,6 +87,19 @@ public function read( &$product ) {
 			clean_post_cache( $product->get_id() );
 		}
 
+		/**
+		 * If the attribute summary is not in sync, update here. Used when searching for variations by attribute values.
+		 * This is meant to also cover the case when global attribute name or value is updated, then the attribute summary is updated
+		 * for respective products when they're read.
+		 */
+		$new_attribute_summary = $this->generate_attribute_summary( $product );
+
+		if ( $new_attribute_summary !== $post_object->post_excerpt ) {
```

Comment:
Can we combine the post_title update and this update into one update, so it's not called twice in some circumstances?

Reply:
Good idea, updated.